FreeRTOS/Test/CMock/smp/single_priority_no_timeslice/single_priority_no_timeslice_utest.c

```diff
@@ -450,8 +450,10 @@ void test_priority_change_tasks_different_priority_lower( void )
     vTaskGetInfo( xTaskHandles[0], &xTaskDetails, pdTRUE, eInvalid );
     TEST_ASSERT_EQUAL( 1, xTaskDetails.xHandle->uxPriority );
 
-    /* Verify the task remains running. When priority dropped in prvSelectHighestPriorityTask.
-     * All the idle core will yield for context switch. The ready queue is a FIFO. Task 0 will be choosed
+    /* Verify the task remains running. */
+    /* When priority dropped in prvSelectHighestPriorityTask, all the idle cores
+     * will yield for context switch. The ready queue is a FIFO. Core 0 will choose
+     * The first task in the ready queue which is task 1. Task 0 will be choosed
```

Comment:
Typo, should be 'chosen' or 'selected'

Reply:
Thanks for the comment. Update in new commit.